### PR TITLE
Feature/#16 nunit testing project

### LINKED ISSUE #16: NUnit testing project
Related to #14

### PR DESCRIPTION
**Pull request by visschersm**
Fixes #16

### CODE REVIEW

**Comment by visschersm** [comment]
Fixes #19

File: tests/Utilities.Test.NUnitTests/NUnitTestExamples.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+#if NET5_0 || NETCOREAPP3_1 || NETFRAMEWORK
+using System;
+#endif
+
+namespace Matr.Utilities.Test.UnitTests
+{
+    public class NUnitTestExamples : TestBase
+    {
+        [SetUp]
+        public void Setup()
```
Comment (pjotrcasteel) [inline tests/Utilities.Test.NUnitTests/NUnitTestExamples.cs:13]: Why?

File: tests/Utilities.Test.NUnitTests/NUnitTestExamples.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        {
+        }
+
+        [Test]
+        public void SimpleExample_WithDependency_ResolvesMockedDependency()
+        {
+            var random = new Random();
+            var returnValue = random.Next(1, int.MaxValue);
```
Comment (pjotrcasteel) [inline tests/Utilities.Test.NUnitTests/NUnitTestExamples.cs:21]: Is leuk maar waarom Random? Voegt toch niks echt toe?

File: tests/Utilities.Test.NUnitTests/NUnitTestExamples.cs (hunk 1 above)
Comment (visschersm) [reply tests/Utilities.Test.NUnitTests/NUnitTestExamples.cs:13]: woops auto generated, I'll remove it :)

File: tests/Utilities.Test.NUnitTests/NUnitTestExamples.cs (hunk 2 above)
Comment (visschersm) [reply tests/Utilities.Test.NUnitTests/NUnitTestExamples.cs:21]: Will always generate different but valid cases.
Though this is more of a demo project so agreed, does not have to be fancy.

Comment (visschersm) [reply tests/Utilities.Test.NUnitTests/NUnitTestExamples.cs]: replaced with ```returnValue = 1```

**Comment by visschersm** [comment]
@pjotrcasteel resolved feedback

**Review by pjotrcasteel — APPROVED** [review]

**Pull request merged**